linux/LinuxProcessTable.c

```diff
@@ -20,7 +20,9 @@ in the source distribution for its full text.
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <syscall.h>
 #include <unistd.h>
+#include <linux/capability.h>
```

Comment:
Shouldn't this be behind a check for when capabilities are actually needed?

Also, this header is covered by an IWYU rule asking to import `sys/capability.h` instead. Or is there explicit reason to use `linux/capability.h` instead here?

Reply:
Fixed the include path.

There is currently no option to disable username coloring for elevated non-root processes, see https://github.com/htop-dev/htop/issues/1379.

Reply:
Reverted the path change, since *<sys/capability.h>* is provided by libcap-dev, which we don't require, and *<linux/capability.h>* is by linux-libc-dev.

Reply:
AFAICS libcap-dev is checked for in `configure.ac` if `--enable-capabilities` is given. And that check looks for `libcap`(-dev) …